packages/fuels-abigen-macro/tests/test_projects/auth_testing_contract/src/main.sw

```diff
@@ -0,0 +1,50 @@
+contract;
```

Comment:
The `fuels-rs` repo probably needs this: https://github.com/FuelLabs/sway/blob/master/.gitattributes for syntax highlighting.